robottelo/ui/base.py

```diff
@@ -879,7 +879,20 @@ def sort_table_by_column(self, column_name):
         ]
         return cell_values
 
-    def perform_action_send_escape_key(self):
-        """Send escape key to browser"""
-        ActionChains(self.browser).send_keys(Keys.ESCAPE).perform()
+    def perform_action_send_keys_to_browser(self, keys):
+        """Send some key(s) to browser.
+
+        :param keys: Key(s) to send to browser. Keys may be several keys
+            including special, like ``control+shift+q``, single special or
+            simple key, e.g. ``escape``, ``q``.
+        """
+        if '+' in keys:
+            keys_to_send = ''.join(
+                getattr(Keys, key.upper()) if key.upper() in dir(Keys) else
+                key for key in keys.split('+'))
+        elif keys.upper() in dir(Keys):
```

Comment:
btw that's kinda weird check. Just use `keys.split('+')` and you'll receive a list of keys which you can process in a single loop, no need in extra elif/else with the same implementation inside for a single key.